packages/botonic-nlp/src/dataset/index.ts

```diff
@@ -0,0 +1,4 @@
+import { Dataset } from './dataset'
+import { Sample } from './types'
+
+export { Dataset, Sample }
```

Comment:
if you want to expose all the exported members within `dataset` and `types`, I think you can also do:

```
export * from './dataset'
export * from './types'
```

Reply:
same for all the other `index.ts`

Reply:
yes, if you want to restrict what to expose, you could do export  { X, Y } from '...' but it's fine to export everything